lib/modules/apostrophe-palette-widgets/public/css/components/fields.less

```diff
@@ -1,13 +1,16 @@
+.apos-palette-ui {
```

Comment:
If this isn't being used directly in templates, it should end with parens, i.e., http://lesscss.org/features/#mixins-feature-not-outputting-the-mixin